docker/openbao-entrypoint.sh

```
@@ -0,0 +1,426 @@
+    eval "value=\$var"
+    if [ -z "$value" ]; then
+      echo "ERROR: Required environment variable $var is not set" >&2
+      exit 1
+    fi
+  done
+
+  PKI_CMD="bao write -field=certificate pki/root/generate/internal \
+    common_name=\"$AM_CERTS_OPENBAO_PKI_CA_CN\" \
+    organization=\"$AM_CERTS_OPENBAO_PKI_CA_O\" \
+    country=\"$AM_CERTS_OPENBAO_PKI_CA_C\" \
+    ttl="$OPENBAO_ROOT_CA_TTL" \
```

Shellcheck (0.11.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)